Web Development/BaseX_Calculator/readme.md

```diff
@@ -0,0 +1,36 @@
+# BaseX Calculator
+
+## Description :-
+
+A calculator that can convert numbers from base-X to base-Y. Example: `` Decimal to Binary ``, `` Binary to Octal ``, `` Base 3 to Base 6 ``, ..etc.
+
+- Convert from & to decimal, binary, hexadecimal, octal.
+- User can go beyond than the known bases and convert from & to other bases such as base-15.
+
+## Tech Stack :-
+
+- HTML
+- CSS
+- Javascript
+
+## Run Local
+
+* Clone this repository
+
+```bash
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts
+```
+
+* Change Directory
+
+```bash
+cd Dev-Scripts/"Web Development/BaseX_Calculator/"
+```
+
+* Open `index.html` in browser.
+
+* Fill in the corresponding fields.
+
+* Click Convert to reveal the result.
+
+## Demonstration [Video](https://drive.google.com/file/d/19hk7j-5EjiTlxYCXNz8pRKqNv7f1LyaB/view?usp=sharing)
```

Comment:
```suggestion
## Demonstration Video
![Video](https://drive.google.com/file/d/19hk7j-5EjiTlxYCXNz8pRKqNv7f1LyaB/view?usp=sharing)
```
![Video](https://drive.google.com/file/d/19hk7j-5EjiTlxYCXNz8pRKqNv7f1LyaB/view?usp=sharing)

or if you want reference then see #53 .

also try to add video in github folder which help us take refrences because if you upload in drive and provide link then if you remove from your drive then it will not visible to us so for safer side upload in GitHub nd make PR . Make this same changes in all your PR till now you  have generated till now.

Reply:
Done.